packages/contracts/test/voting/multisig.ts

```diff
@@ -99,6 +99,20 @@ describe('Multisig', function () {
     );
   });
 
+  function addresslist(length: number): string[] {
+    let addresses: string[] = [];
+
+    for (let i = 0; i < length; i++) {
+      const addr = signers[i].address;
+      addresses.push(addr);
+    }
+    return addresses;
+  }
+
+  async function timestampIn(durationInSec: number): Promise<number> {
+    return (await ethers.provider.getBlock('latest')).timestamp + durationInSec;
+  }
+
```

Comment:
Probably, we should create a `time-helpers.ts` class.
 `packages/contracts/test/test-utils/voting.ts` also contains some functions (that could use improvement regarding the usage of `await`).

Reply:
done